### DISCUSSION TITLE
Cannot run this cmd: brew install planetscale/tap/pscale

### DISCUSSION THREAD

**Discussion by vipoooooo** [planetscale/discussion#231] [Q&A] [answered]
brew install planetscale/tap/pscale
Error: Can't create update lock in /usr/local/var/homebrew/locks!
Fix permissions by running:
  sudo chown -R $(whoami) /usr/local/var/homebrew
==> Tapping planetscale/tap
fatal: could not create leading directories of '/usr/local/Homebrew/Library/Taps/planetscale/homebrew-tap': Permission denied

**Comment by fooforge** [comment, marked as answer]
This looks more like an issue with your homebrew installation than with our tap. What's `brew config` and `brew doctor` saying on your client?

**Reply by vipoooooo** [reply]
> brew config 
HOMEBREW_VERSION: 3.3.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 96137bc19e68398ebbb7033379df288cd8b9a3f9
Last commit: 7 months ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: bbc06ae41bec4dbf409f98fddedfc38df2fd7d48
Core tap last commit: 6 months ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: 10-core 64-bit westmere
Clang: 13.1.6 build 1316
Git: 2.23.0 => /usr/local/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.2.1-x86_64
CLT: N/A
Xcode: 13.4.1
Rosetta 2: true





> brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!
Warning: The following directories are not writable by your user:
/usr/local/etc/bash_completion.d
/usr/local/share/info
/usr/local/share/zsh
/usr/local/share/zsh/site-functions
/usr/local/var/log
You should change the ownership of these directories to your user.
  sudo chown -R $(whoami) /usr/local/etc/bash_completion.d /usr/local/share/info /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var/log
And make sure that your user has write permission.
  chmod u+w /usr/local/etc/bash_completion.d /usr/local/share/info /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var/log
Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.
(and alot of file directory that i didn't paste here)

**Reply by vipoooooo** [reply]
I don't know what all of these mean but these are what it said

**Reply by fooforge** [reply]
Thanks!

Yeah, so as already thought this is an issue with your Homebrew installation and not with our CLI tool `pscale`. You will need to follow the steps that `brew doctor` printed out (fix directory ownership, ensure your user has write permission, remove _unbrewed_ header files, and so on).

If these steps aren't enough to fix the problem, make sure you read through Homebrew's documentation [on brew.sh](https://brew.sh) and if that does not help either, you can find their community discussion board [here](https://github.com/Homebrew/discussions/discussions). However, make sure to search their archive before posting anything new. It is a fairly common problem after all, and you should be able to resolve this on your own by doing a bit of reading. Good luck!
